tests/cache_file_generator.py

```diff
@@ -1,3 +1,11 @@
+"""
+This is a console application which is to be used for cross-platform lock performance testing.
+The app will acquire lock for the cache file, log the process id and free the lock
```

Comment:
```suggestion
The app will acquire lock for the cache file, log the process id and release the lock
```

I believe the jargon is acquire and release.

By the way, since you go this far, you might as well do [this](https://stackoverflow.com/questions/7791574/how-can-i-print-a-python-files-docstring-when-executing-it), to also help this CLI tool's user.

Reply:
Nice suggestion, love the idea mentioned in the link, making that change.